uaclient/contract.py

```diff
@@ -208,10 +206,6 @@ def get_resource_machine_access(
             )
         if response.headers.get("expires"):
             response.json_dict["expires"] = response.headers["expires"]
-        if save_file:
-            self.cfg.write_cache(
-                "machine-access-{}".format(resource), response.json_dict
-            )
```

Comment:
Just making sure - anbox doesn't rely on this right? We write the credentials in another way for them to use?

Reply:
No, it doesn't. We are storing the tokens for them in a separate file